components/grid/export/events.md

```diff
@@ -251,6 +254,135 @@ To export a hidden Grid column that has its `Visible` parameter set to `false`,
 }
 ````
 
+### For PDF Export
+
+* `Columns` - `List<GridPdfExportColumn>` - a collection of all exportable columns in the Grid. These are all visible `GridColumn` instances. You can customize the following attributes of the Grid column before exporting it into PDF:
+
+    * `Width` - define the width of the column **in pixels**.
+    * `Title` - define the column title to be shown in the PDF file header. 
+    * `NumberFormat` - provide an PDF-compatible number/date format
```

Comment:
In the case of abbreviations, choosing between a/an depends on the pronunciation of the abbreviations. In this case, it would be `a pi-di-ef`. But with svgs, for examples, it would be `an es-vi-dʒi`

```suggestion
    * `NumberFormat` - provide a PDF-compatible number/date format.
```